src/main.rs

```diff
@@ -177,7 +177,8 @@ impl FlockingPlugin {
                 }),
                 translation: Translation::new(0.0, 0.0, 0.0),
                 sprite: Sprite {
-                    size: Vec2::new(window.width as f32, window.height as f32)
+                    size: Vec2::new(window.width as f32, window.height as f32),
+                    resize_mode: SpriteResizeMode::Automatic,
```

Comment:
```suggestion
                    ..Default::default()
```

might be more resilient to future changes to the SpriteComponent API